bin/g5-calc-noise.sh

```diff
@@ -87,26 +88,26 @@ do
   # calculate sum of distances (all deltas) 
   sod=$(bc <<< "$sod + $absDelta")
 
-  # Any single jump amount over 18 increases noise linearly
-  remainder=$(bc <<< "$absDelta - 18")
+  # Any single jump amount over a certain limit increases noise linearly
+  remainder=$(bc <<< "$absDelta - $RISE_WITHOUT_ADDED_NOISE")
   if [ $(bc <<< "$remainder > 0") -eq 1 ]; then
     # noise higher impact for latest bg, thus the smaller denominator for the remainder fraction 
     noise=$(bc -l <<< "$noise + $remainder/(300 - $i*30)") 
   else
     remainder=0
   fi
   
-  #echo "lastdelta=$lastDelta, delta=$delta, remainder=$remainder, 
+  #echo "lastdelta=$lastDelta, delta=$delta, remainder=$remainder" 
   #echo "sod=$sod, noise=$noise, absDelta=$absDelta"
 
   lastDelta=$delta
 done
 
 # to ensure mostly straight lines with small bounces don't give heavy noise
-  if [ $(bc -l <<< "$noise > $LIGHT_MAX_NOISE") -eq 1 ]; then
-    if [ $(bc -l <<< "($sod / $n) < 3") -eq 1 ]; then
+  if [ $(bc -l <<< "$noise > $CLEAN_MAX_NOISE") -eq 1 ]; then
+    if [ $(bc -l <<< "($sod / $n) < 6") -eq 1 ]; then
```

Comment:
do you want to set a constant for this magic value? i'm not sure what it represents...

Reply:
Yes, I'll call them CLEAN_MAX_AVERAGE_DELTA, and LIGHT_MAX_AVERAGE_DELTA